src/main/kotlin/it/pagopa/wallet/eventdispatcher/queues/WalletCdcQueueConsumer.kt

```diff
@@ -60,19 +60,18 @@ class WalletCdcQueueConsumer(
     private fun handleCdcEvent(event: LoggingEvent): Mono<Unit> {
 
         return tracing.customizeSpan(
-            if (
-                event is WalletOnboardCompletedEvent ||
-                    event is WalletDeletedEvent ||
-                    event is WalletApplicationsUpdatedEvent
-            ) {
-                Mono.just(
-                    logger.info(
-                        "Process event with id {} of type {} published on {}",
-                        event.id,
-                        event.type,
-                        event.timestamp
-                    )
-                )
+            if (event is WalletLoggingEvent) {
+                Mono.defer { walletCDCService.sendToKafka(event) }
```

Comment:
Please consider calling sendToKafka method directly since a Mono.defer is already in the method's implementation

Reply:
sure, done here d9616ced92e72ffa52bca44fb01850c063e04675